src/test/solution_space/solution_space_handler/find_same_fragments_test.py

```diff
@@ -1,11 +1,12 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
 import ast
+import pytest
```

Comment:
Pytest is not a standard library, so please, move this import into a separated second group